src/main/java/org/terasology/structureTemplates/util/BlockRegionUtilities.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.structureTemplates.util;
+
+
+import org.terasology.math.Region3i;
+import org.terasology.math.geom.Vector3f;
+import org.terasology.math.geom.Vector3i;
+import org.terasology.structureTemplates.components.SpawnBlockRegionsComponent;
+import org.terasology.structureTemplates.util.transform.BlockRegionMovement;
+
+import java.util.List;
+
+
+/**
+ * A collection of little helper to ease the work with structure templates.
+ **/
+
```

Comment:
Oh the empty line was by accident. 
As for being more specific: I guess I adding some method descriptions instead of that line could be more helpful then.